syntax = "proto3";

package hello;

service HelloGRPC{
      rpc SayHello (HelloRequest) returns (HelloReply);
}

message HelloRequest {
      string name = 1;
}

message HelloReply{
    bool status = 1;
    string msg = 2;
} 
